Installers/Electron/src/utils.js

```diff
@@ -73,8 +73,8 @@ class Utils {
                 });
                 return path;
             },
-            "darwin": () => "/Applications/Discord.app",
-            "linux": () => "" // TODO
+            "darwin": () => "/Applications/Discord.app/Contents",
+            "linux": () => "/usr/share/discord-canary"
```

Comment:
Is `/usr/share/discord-canary` used on every distribution?
The AUR packages is using `/opt/discord-canary` for instance, not sure if that's the case in other places.